examples/bbox_detection/labelme2voc.py

```diff
@@ -101,7 +106,19 @@ def main():
             class_id = class_names.index(class_name)
 
             (xmin, ymin), (xmax, ymax) = shape['points']
-            bboxes.append([xmin, ymin, xmax, ymax])
+            
+            # swap if min is larger than max.
+            if xmin > xmax:
+                tmp = xmin
+                xmin = xmax
+                xmax = tmp
+            if ymin > ymax:
+                tmp = ymin
+                ymin = ymax
+                ymax = tmp
+            
+            # need to write as int
+            bboxes.append([int(xmin), int(ymin), int(xmax), int(ymax)])
```

Comment:
I would say -1 for int casting. Because it just loses precision and I'm not sure why it is necessary.

Reply:
Did INT casting since the voc parser I was using only recieved int bbox information.
Since this is my specifc problem, int casting was removed and people can add this if necessary.